src/main/java/seedu/address/model/person/Person.java

```diff
@@ -180,13 +183,25 @@ public DateOfBirth getDateOfBirth() {
         return dob.get();
     }
 
+    //@@author arnollim
     @Override
     public String getReason() {
         Address a = this.getAddress();
         Name n = this.getName();
-        this.reason = String.format(SHOWING_WHY_MESSAGE, n, a);
```

Comment:
Your var names are screaming at me, please go read the textbook on naming conventions, only ever use single letter for dummy vars, IDE autofills for you so why u being lazy?

Reply:
fixed